DOC-3785 updated clients landing page (#228)
redis/docs: content/develop/connect/clients/_index.md

weight: 45
---

Use the Redis client libraries to connect to Redis servers from

> your own code. We currently support and document client libraries
> for 4 main languages:

Reviewer: Two style guide violations: (1) "...currently...", see [here](https://developers.google.com/style/word-list#currently); (2) change "4" to "four", see [here](https://developers.google.com/style/numbers).

```suggestion
your own code. We support and document client libraries
for four main languages:
```

Author: Done, and there was another "currently" elsewhere.

[Python]({{< relref "/develop/connect/clients/python" >}}),
[C#/.NET]({{< relref "/develop/connect/clients/dotnet" >}}),
[Node.js]({{< relref "/develop/connect/clients/nodejs" >}}),
and [Java]({{< relref "/develop/connect/clients/java" >}}).
We also provide higher-level
[object mapping (OM)]({{< relref "/develop/connect/clients/om-clients" >}})
for these languages.

We also develop libraries for the following languages but with limited
support and documentation on external sites:

- [Go](https://redis.uptrace.dev/guide/)
- [Ruby](https://github.com/redis/redis-rb)
- [C](https://github.com/redis/hiredis)

> To use the client libraries, you will need a Redis server to connect to.

Reviewer: ```suggestion
You will need a Redis server to use these libraries.
```

Author: Done.

You can experiment with a local installation of Redis Stack
(see [Install Redis Stack]({{< relref "/operate/oss_and_stack/install/install-stack/" >}})) or with a free trial of [Redis Cloud]({{< relref "/operate/rc" >}}).

> To interact with a Redis server without writing code, use our

Reviewer: ```suggestion
To interact with a Redis server without writing code, use
```

Author: Done.
